main.py

```diff
@@ -0,0 +1,63 @@
+from robyn import Robyn, SubRouter, jsonify
+import json
+
+mainApp = Robyn(__file__)
+
+def ex1():
+    print("working ex1")
+    return 999
+def ex2():
+    print("working ex2")
+    return 667
+
+class Example3:
+    def __init__():
+        '''print("Initialized")
+        return "init initialized and returned"'''
+    def ex3_method():
+        print("working example 3 method printed")
+        return "working example 3 returned"
+
+mainApp.inject("/s",example1=ex1) #specified inject, if you set to =ex1(), it will point to int return object
+mainApp.inject(example2=ex2) #unspecified inject
```

Comment:
The inject function should allow multiple dependency injections in one call.

e.g. `mainApp.inject(example1=ex1, example2=ex2, ...)`

Reply:
Hello @sansyrox, just addressed that in the most recent commit

Reply:
Hello @sansyrox Would you mind providing some unit test cases to multi-route testing?